src/contexts/SiteLaunchContext.tsx

```diff
@@ -0,0 +1,87 @@
+import { createContext, useContext, useEffect, useState } from "react"
+
+import { useGetSiteLaunchStatus } from "hooks/siteDashboardHooks"
+
+import {
+  SiteLaunchDto,
+  SiteLaunchFrontEndStatus,
+  SiteLaunchStatusProps,
+  SITE_LAUNCH_TASKS_LENGTH,
+} from "types/siteLaunch"
+
+interface SiteLaunchContextProps {
+  setSiteName: (siteName: string) => void
```

Comment:
Do we need a hook for this? Will we be unable to get it from `useParams<{ siteName: string }>()`?

Reply:
Right yes this is true, thanks for this!
dfd54ce12e139b20cd5a0e955ac63b72e513814a